packages/react-native/ReactAndroid/src/main/java/com/facebook/react/module/model/ReactModuleInfo.java

```diff
@@ -7,6 +7,8 @@
 
 package com.facebook.react.module.model;
 
+import com.facebook.react.turbomodule.core.interfaces.TurboModule;
```

Comment:
Shouldn't we be using the `.internal` one? This is what tripped me up as well (and likely others). It'd better if we could make this a build time break instead e.g. if `com.facebook.react.turbomodule.core.interfaces.TurboModule` was removed entirely.
```suggestion
import com.facebook.react.internal.turbomodule.core.interfaces.TurboModule;
```

Reply:
Alternatively, I think you can solve this by using `typealias`? In `com.facebook.react.turbomodule.core.interfaces.TurboModule`:
```kotlin
typealias TurboModule = com.facebook.react.internal.turbomodule.core.interfaces.TurboModule
```

Reply:
> Shouldn't we be using the .internal one?

Yup we should OR and check both classes actually